assam_budget_board/templates/home.html

```diff
@@ -0,0 +1,98 @@
+{% extends "page.html" %}
+{% load cms_tags menu_tags staticfiles  %}
+{% block css_files %} 
+    {{ block.super }}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/home.css' %}">
+{% endblock css_files %} 
+
+{% block title %}{% page_attribute "page_title" %}{% endblock title %}
+
+{% block navbar %}
+<nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav">
+        <div class="container">
+            <a class="navbar-brand js-scroll-trigger" href="#page-top">Assam State Budget Board</a>
+            <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                <i class="fa fa-bars"></i>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarResponsive">
+                <ul class="navbar-nav ml-auto">
+                    {% show_menu 0 1 1 1 "menu/home_menu.html" %}
+                </ul>
+            </div>
+        </div>
+    </nav>
+{% endblock navbar %}
```

Comment:
is the navbar going to be updated by any child template? if not then we could directly keep the code here without the block?

Reply:
Yes. The Navbar for Homepage and rest of the pages would be different thus, the block.